TFT/src/User/config.ini

```diff
@@ -522,11 +543,11 @@ preheat_temp6:T250 B90
 
 
 #--------------------------------------------------------------------
-# Power Supply Settings (if connected to TFT controller)
+# Power Supply Settings (if connected to TFT controller only)
 #--------------------------------------------------------------------
 
 #### Default Power Supply Mode
-# Enable auto shutdown when hot-end temperature becomes below target value.
+# Enable auto shutdown when hotend temperature becomes below target value.
```

Comment:
"falls below target value", "drops below target value" or "becomes lower than the target value"